CHANGELOG_PENDING.md

```diff
@@ -1,7 +1,26 @@
 ### Improvements
 
+- [cli] - Fix the preview experience for unconfigured providers. Rather than returning the
+  inputs of a resource as its outputs, the engine will treat all outputs as unknown. Most
```

Comment:
It feels like theres a qualifier missing on this sentence?  Something like this?

```suggestion
  inputs of a resource managed by an unconfigured provider as its outputs, the engine will treat all outputs as unknown. Most
```

Reply:
Sure--I thought that the prior sentence would imply this.